style.css

```diff
@@ -701,4 +822,60 @@ fieldset > * {
   .blackline {
     display: none;
   }
+
+  .firstcardcontainer {
+    display: flex;
+    gap: 10px;
+    flex-direction: column;
+    justify-content: space-between;
+    background-color: #fff;
+    padding: 20px 16px;
+    border-radius: 16px;
+    width: fit-content;
+    position: relative;
+    opacity: 1;
+  }
+
+  .popup {
+    display: none;
+    height: 120vh;
+    background: #c1c7d0;
+    opacity: 0.8;
+  }
```

Comment:
- [ ] Nice work implementing the popup menu :clap: However it would be nice to remove the `opacity` property for `.popup` to follow the Figma design
### Your design :point_down: 
![Screenshot from 2023-06-21 18-13-38](https://github.com/anshuman7negi/my-portfolio/assets/91142432/4a82729a-01b0-4237-9c64-95f86c69ecf3)
### What it's supposed to look like :point_down: 
![Screenshot from 2023-04-19 13-38-59](https://github.com/anshuman7negi/my-portfolio/assets/91142432/47d40597-1676-41d8-925d-7db305694c02)